openslides_backend/models/motion_state.py

```diff
@@ -0,0 +1,31 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionState(Model):
+    """
+    Model for motion state.
+
+    Reverse fields:
+    TODO
+    """
+
+    collection = Collection("motion_state")
+    verbose_name = "motion_state"
+
+    id = fields.IdField(description="The id of this motion state.")
+    name = fields.RequiredCharField(description="The name of this motion state.")
+    recommendation_label = fields.CharField(
+        description="The recommendation label of thie motion state."
+    )
+    css_class = fields.CharField(description="The css class of thie motion state.")
+    # TODO restrictions: string[],
+    # TODO allow_support: boolean;
+    # TODO allow_create_poll: boolean;
+    # TODO allow_submitter_edit: boolean;
+    # TODO set_number: boolean;
+    # TODO show_state_extension_field: boolean;
+    # TODO merge_amendment_into_final: number;
+    # TODO show_recommendation_extension_field: boolean;
```

Comment:
In #101 I added `BooleanField` and `ArrayField` so you can implement these fields now. For business logic (which fields can be set by user on create/update, @FinnStutzenstein should probably weigh in again ;)

Reply:
Merged #101 and added these fields.